README.md

```diff
@@ -40,6 +40,12 @@ A synchronized element's content can also be based on concatenating multiple `<k
 
 To add new lines between items, use `'\n'` for example `sync:<key1> + '\n' + <key2>`.
 
+## Content wildcards
+
+A synchronized element's content can be chosen by pattern. For example
```

Comment:
OK, sounds good. It seems like the main feature here is for wildcard uses. I think this is a great first go at it.